test/constructors.jl

```diff
@@ -23,13 +23,13 @@ const ≅ = isequal
     @test size(df, 1) == 3
     @test size(df, 2) == 2
 
-    df2 = DataFrame!(collect(Any, vecvec), Index([:x1, :x2]))
+    df2 = DataFrame(collect(Any, vecvec), Index([:x1, :x2]), copycols=false)
     @test size(df2, 1) == 3
     @test size(df2, 2) == 2
     @test df2.x1 === vecvec[1]
     @test df2.x2 === vecvec[2]
 
-    for fun in (DataFrame, DataFrame!)
+    for fun in (DataFrame, (args...; kwargs...) -> DataFrame(args...; kwargs..., copycols=false))
```

Comment:
It would be simpler to do `for copycols in (true, false)` and pass the argument directly..

Reply:
right - I will change it